### ISSUE TITLE
Obtaining Phoneme Logits

### ISSUE THREAD

**Issue by konan-ai** [m-bain/whisperX#341]
I'm attempting to tap into the WhisperX framework to extract predicted phoneme logits prior to their alignment. However, I'm currently uncertain as to where exactly in the codebase this information might reside.

One section of code that I've come across in the alignment.py file seems potentially relevant, but I would appreciate some confirmation:

https://github.com/m-bain/whisperX/blob/befe2b242eb59dcd7a8a122d127614d5c63d36e9/whisperx/alignment.py#L82C9-L82C49

```python
align_model = align_model.to(device)
labels = processor.tokenizer.get_vocab()
```

I'm not entirely convinced that this is the correct spot to obtain the unaligned logits. If anyone could provide insight on whether this is the right direction, or if there's a more appropriate place to look, it would be greatly appreciated.

Thanks in advance for your help!
